### ISSUE TITLE
Optimal settings

### ISSUE THREAD

**Issue by rvandemaat** [andrastoth/webcodecamjs#37]
Dear mr. Toth,

thank you for your QR- and Barcodescanner.
I've implemented it in an Angular2 environment.

It all works fine, but there is one issue. When scanning QR codes, the scanner doesn't scan in an optimal way. It takes a while before the scanner scans the QR code and sometimes it doesn't scan at all (barcodes are scanned fine by the way).

I think I don't use the right settings. So my question is: what are the optimal settings for scanning a QR code?

Greetings,
Rik van de Maat

**Comment by andrastoth** [comment]
Hello,

It is very strange because the qr reader works better.
Please check:
- image border must be white border,
- image must be sharp (focus range)

I using these settings:

```
var args = {
        autoBrightnessValue: 100,
        decodeQRCodeRate: 5,
        decodeBarCodeRate: 5,
        resultFunction: function(res) {

        }
    };

```

Regards
András

**Comment by tearforfear007** [comment]
@rvandemaat Hi, can I know how to implement this scanner in the Angular2 environment?

**Comment by rvandemaat** [comment]
@tearforfear007 
Hi,

what I did were the follow steps:
1. I setup the webcodecam code in a Angular 2 directive (https://angular.io/guide/attribute-directives).
2. Because I needed it in my project more than once at the same time, I choose to make a service (https://angular.io/tutorial/toh-pt4) in which I instantiate the webcodecam directive.
3. Then I made a component, with template and all, in which I injected the service.

It then worked fine for me.
